stonesoup/types/prediction.py

```diff
@@ -142,13 +142,27 @@ class ParticleStatePrediction(Prediction, ParticleState):
     """
 
 
+class PointMassStatePrediction(Prediction, PointMassState):
+    """ParticleStatePrediction type
+
+    This is a simple Particle state prediction object.
+    """
```

Comment:
This docstring is for ParticleStatePrediction, could you make a minor change to make it relevant for PointMassStatePrediction ?

Reply:
I am also lost what at what is the purpose of the prediction object, but everything seems to be working....